2-Draft-Accepted/RFCxxxx-DSC-for-PowerShell7.md

```diff
@@ -0,0 +1,323 @@
+  resources and configurations
+- the `PSDesiredStateConfiguration` module that includes cmdlets like `Get-DscResource`
+
+DSC ships as a component of Windows 10 and as an open source project for Linux.
+It is also integrated into cloud-based server management services in multiple
+public clouds including Microsoft Azure and Amazon Web Services (AWS).
+
+The versions of DSC that ship in Windows PowerShell are tightly coupled with
+[Windows Management Instrumentation (WMI)](https://docs.microsoft.com/windows/win32/wmisdk/wmi-start-page)
+on Windows and
+[Open Management Interface](https://www.dmtf.org/content/open-management-interface-omi-%E2%80%93-open-source-implementation-dmtf-cimwbem-standards)
+(OMI) on Linux.
+The *MI libraries are used to host the Local Configuration Manager (LCM) and APIs
+are used to parse information store in Managed Object Framework (MOF) file format,
+which is used for both configuration files and schema files.
+
+### Challenges with the current versions of DSC
+
+Very few community authored DSC resources exist for Linux because C++ is the only
+supported language. Other than using the Script/nxScript resources combined with
+a composite resource, it is near-impossible today to create a DSC solution that is cross-platform.
+
+Today, third party solutions leveraging DSC resources need to call the LCM
+to run Get/Set/Test, making it complicated to integrate with partner ecosystems.
+
+## Scope of DSC implementation for PowerShell 7
+
+Unlike DSC for Windows PowerShell, DSC for PowerShell 7 is only intended to provide
+a platform for building or integrating with configuration solutions. For PowerShell 7,
+the scope of DSC support is limited to the `configuration` keyword, authoring,
+compilation, and using DSC resources via the PSDesiredStateConfiguration module.
+
+This means that a default agent (LCM) is not included and existing pull servers are not supported.
+
+### Motivation
+
+DSC in PowerShell 7 should enable the following user scenarios:
+
+    As a developer of configuration management solutions,
+    I can easily integrate DSC resources
+    so that my solution can leverage the PowerShell ecosystem and its users and contributors.
+
+    As an IT Pro,
+    I can directly invoke DSC resources
+    so that I can build ad hoc cross platform configurations.
+
+### Cross platform first
+
+DSC has supported both Windows and Linux, however, the
+[Linux support](https://docs.microsoft.com/powershell/scripting/dsc/getting-started/lnxGettingStarted)
+was dependent on a Microsoft-maintained implementation of
+[OMI](https://github.com/Microsoft/omi)
+and did not encourage authoring Linux DSC resources in PowerShell,
+making development inconsistent and difficult for non-expert users.
+
+DSC for PowerShell 7 will prioritize being cross platform and having a consistent
+user experience and capabilities on all supported platforms.
+
+### Invoke-DscResource
+
+About a year ago, we worked with
```

Comment:
Although i appreciate the shout-out to Gael, I'm not sure this paragraph belongs here. Perhaps an acknowledgement section in the back of the RFC would be more appropriate.
I think what we should have here is a description of the behavior that we will be supplying.

Reply:
The point here was to connect it back to the `Invoke-DscResource` work